solvation_analysis/tests/test_plotting.py

```diff
@@ -0,0 +1,51 @@
+import numpy as np
+import pytest
+
+from solvation_analysis.plotting import (
+    plot_network_size_histogram,
+    plot_shell_size_histogram,
+    compare_pairing,
+    compare_coordination_numbers,
+    compare_residence_times,
+    compare_speciation,
+    format_graph,
+)
+
+
+def test_plot_network_size_histogram(networking):
+    fig = plot_network_size_histogram(networking)
+    fig.show()
```

Comment:
I don't think you ever need to have `show()` in library code (and certainly not in the tests): either you can return the fig object and then the user can do what they want (including show) or your code plot to files and generates files (and then it slows you down a lot to run show).